gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/CredentialsController.groovy

```diff
@@ -78,4 +90,49 @@ class CredentialsController {
                             @RequestHeader(value = "X-RateLimit-App", required = false) String sourceApp) {
     return getAccountDetailsWithAuthorizedFlag(user).find { it.name == account }
   }
+
+  @GetMapping
```

Comment:
I apparently don't know enough about Spring WebMVC to figure out how this endpoint conflicts with either of the existing ones. This is causing a test error (and startup failure).